Makefile

```diff
@@ -112,6 +112,9 @@ test:
 clean:
 	go clean
 	rm ./bin/${BINARY_NAME}
+	
+clean-crl-server:
```

Comment:
curious where it's used

Reply:
This is for negative cases. Code base updated.